style.css

```diff
@@ -67,12 +67,11 @@ body {
 
 .close-icon {
   display: none;
-  background: #6070ff;
   color: #fff;
 }
 
 .menu {
```

Comment:
- [ ] You did on implementing the changes requested by the previous reviewer 👍️ but, still, the background is not blurred. 
Adding this  👉️`backdrop-filter: blur(5px);`  to your `menu` class should do the work.

Reply:
@ridwanediallo I added and still does not look blurred.
Please can you send me your mail so I send for you to take a look

Reply:
Thank you very much, it worked